workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -9,6 +9,14 @@ params.annotation_dir = 'annotation'
 params.t2g = 'Homo_sapiens.GRCh38.103.spliced.tx2gene.tsv'
 params.mitolist = 'Homo_sapiens.GRCh38.103.mitogenes.txt'
 params.sketch = false // use sketch mode for mapping with flag `--sketch`
+params.resolution = 'full' //default resolution is full, can also use cr-like, cr-like-em, parsimony, and trivial
```

Comment:
We might want to add an error check in the workflow to be sure that this value is valid before we get to the `quant_fry` process.

Reply:
Definitely a good idea. For now I'm going to agree that this is something we can add in in the future if we end up doing more with this workflow. Right now, it will not run without that parameter being valid, but it would be good to check for any potential errors before running the other steps.